Updated 02_Statistics

abr-98 opened the pull request:
Updated:

- EDA
- Histograms
- Percentiles and Histograms
- Probability Theory
- Bayes Theorem
- Random Variables
- Skewness

MrMimic commented on 02_Statistics/README.md:
```diff
@@ -50,22 +50,188 @@ The median is the value __separating the higher half of a data sample, a populat
+
+__Installation__:
+
+    sudo pip3 install pandas
```
No installation with sudo. Better to create a venv first.

abr-98 replied:
Yeah!! I missed that. Using "sudo" will install to the root. It's better to use a venv. Thanks

MrMimic commented on 02_Statistics/README.md:
```diff
@@ -50,22 +50,188 @@ The median is the value __separating the higher half of a data sample, a populat
+
+__Installation__:
+
+    sudo pip3 install seaborn
```
Idem here

abr-98 replied:
I have resolved the issue

MrMimic replied:
Sorry, was busy !

Thanks for this nice update !

MrMimic reviewed (changes requested):
That's a great PR ! Thanks ! Juste few comments about the use of "sudo" while installling Python packages.

abr-98 commented:
Updated:

- 8_ Random variables
- 9_ Cumul Dist Fn (CDF)
- 10_ Continuous distributions
- 12_ ANOVA
- 13_ Prob Den Fn (PDF)
- 16_ Hypothesis Testing
- 17_ p-Value
- 18_ Chi2 test
- 22_ Kernel Density estimate
- 23_ Regression
- 24_ Covariance
- 25_ Correlation
- 29_ Euclidian Distance

abr-98 commented:
Updated  04_Machine-Learning

updated:

- 2_ Numerical var
- 3_ Categorical var
- 4_ Supervised learning
- 5_ Unsupervised learning
- 6_ Concepts, inputs and attributes
- 7_ Training and test data
- 8_ Classifiers
- 9_ Prediction
- 10_ Lift
- 11_ Overfitting
- 12_ Bias & variance
- 13_ Tree and classification
- 14_ Classification rate
- 15_ Decision tree
- 16_ Boosting

abr-98 commented:
Added Reinforcement Learning

abr-98 commented:
Updated:

- 17_ Naïves Bayes classifiers
- 18_ K-Nearest neighbor
- 19_ Logistic regression
- 20_ Ranking
- 21_ Linear regression
- 23_ Hierarchical clustering
- 24_ K-means clustering
- 25_ Neural networks
- 26_ Sentiment analysis
- 27_ Collaborative filtering
- 28_ Tagging
- 29_ Support Vector Machine
- 30_Reinforcement Learning
